sdk/src/wallet/operations/syncing/options.rs

```diff
@@ -40,9 +40,9 @@ pub struct SyncOptions {
     /// Checks pending transactions and reissues them if necessary.
     #[serde(default = "default_sync_pending_transactions")]
     pub sync_pending_transactions: bool,
-    /// Specifies what outputs should be synced for the ed25519 addresses from the account.
+    /// Specifies what outputs should be synced for the ed25519 addresses from the wallet.
     #[serde(default)]
-    pub account: AccountSyncOptions,
+    pub wallet: WalletSyncOptions,
     /// Specifies what outputs should be synced for the address of an account output.
     #[serde(default)]
     // TODO Rename when we are done with Account changes https://github.com/iotaledger/iota-sdk/issues/647.
```

Comment:
Idk if you want to, but since you renamed `AccountSyncOptions`, we're able to address this TODO

Reply:
Done